docs/quickstarts/nextjs.mdx

```diff
@@ -74,39 +75,39 @@ All Clerk hooks and components must be children of the [`<ClerkProvider>`](/docs
 Select your preferred router below to learn how to make this data available across your entire app:
 
 <CodeBlockTabs type="router" options={["App Router", "Pages Router"]}>
-    ```tsx filename="/src/app/layout.tsx" {1,10-14}
-    import { ClerkProvider } from '@clerk/nextjs'
-    import './globals.css'
-
-    export default function RootLayout({
-      children,
-    }: {
-      children: React.ReactNode
-    }) {
-      return (        
-        <ClerkProvider>
-          <html lang="en">
-            <body>{children}</body>
-          </html>
+  ```tsx filename="/src/app/layout.tsx" {1,10-14}
+  import { ClerkProvider } from '@clerk/nextjs'
+  import './globals.css'
+
+  export default function RootLayout({
+    children,
+  }: {
+    children: React.ReactNode
+  }) {
+    return (        
+      <ClerkProvider>
+        <html lang="en">
+          <body>{children}</body>
+        </html>
+      </ClerkProvider>
+    )
+  }
+  ```
+
+  ```tsx filename="_app.tsx" {2,6-8}
+    import '@/styles/globals.css'
+    import { ClerkProvider } from "@clerk/nextjs/server";
+    import type { AppProps } from "next/app";
+    function MyApp({ Component, pageProps }: AppProps) {
+      return (          
+        <ClerkProvider {...pageProps}>
+          <Component {...pageProps} />
         </ClerkProvider>
-      )
+      );
     }
+    export default MyApp;

```

Comment:
formatting stuff again :/

Reply:
This is just being applied when I save. It might be a prettier thing of some sort?